app/vue3/tsconfig.json

```diff
@@ -6,7 +6,8 @@
       "webpack-env",
       "node"
     ],
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "skipLibCheck": true
```

Comment:
This package also seems to be having a bad time with JSX typings, skipping for now.